selection_tools/src/Particle.cpp

```diff
@@ -105,6 +106,13 @@ bool Particle::is_final()
   return true;
 }
 
+bool Particle::is_prompt()
+{
+  if(sqrt(pow(x, 2) + pow(y, 2)) > 0.2) return false;
+
+  return true;
```

Comment:
You could simplify that to `return sqrt(pow(x, 2) + pow(y, 2)) <= 0.2;`.
Oh, and by the way just a stupid comment: `x*x` is shorter and maybe even cleaner than `pow(x, 2)` :)